docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -44,81 +39,186 @@ Specifically, this guide will have you do the following:
 
```

Comment:
What about a paragraph here saying what the cluster will look like - i.e. it will be conformant meaning it will have DNS, SSL etc...